pkg/source/image.go

```diff
@@ -138,13 +138,12 @@ func (i *Image) getDesiredPodApplyConfig(bundle *rukpakv1alpha2.BundleDeployment
 				WithName("install-unpacker").
 				WithImage(i.UnpackImage).
 				WithImagePullPolicy(corev1.PullIfNotPresent).
-				WithCommand("cp", "-Rv", "/unpack", "/util/bin/unpack").
+				WithCommand("/cp", "-Rv", "/unpack", "/util/bin/unpack").
 				WithVolumeMounts(applyconfigurationcorev1.VolumeMount().
 					WithName("util").
 					WithMountPath("/util/bin"),
 				).
-				WithSecurityContext(containerSecurityContext).
-				WithTerminationMessagePolicy(corev1.TerminationMessageFallbackToLogsOnError),
```

Comment:
I thought @joelanford had just put these in? Why are they being removed?

Reply:
https://github.com/operator-framework/rukpak/pull/872

Reply:
They got stomped from the file that was in op-con, fixing now :+1: